src/metabase/db/custom_migrations.clj

```diff
@@ -1644,7 +1644,7 @@
                       :channel_type "channel/email"
                       :details      (json/generate-string {:type           "email/mustache-resource"
                                                            :subject        "Your Slack connection stopped working"
-                                                           :path           "metabase/email/slack_token_error.mustache"
+                                                           :path           "metabase/email/slack_token_error"
```

Comment:
Is there not another extension that makes sense?

Reply:
yea we no longer need the extension